src/components/common/Card/index.jsx

```diff
@@ -0,0 +1,25 @@
+import {
+  ProductContainer,
+  ImageContainer,
+  ProductName,
+  ProductPrice,
+  ProductImage,
+} from './styles'
+
+export const ProductCard = ({ name, price, productImage }) => {
+  return (
+    <div>
```

Comment:
Incluir a tag <Link> com rota dinâmia para o produto selecionado. Exemplo 

// <Link to={`${BaseUrl}/produto/${id}`}>Clique aqui</Link>